docs/usage.rst

```diff
@@ -387,11 +387,16 @@ By extending the event with subcomponents, you can create multiple alarms.
     >>> alarm_24h_before.add('description', 'Reminder: Event in 24 hours')
     >>> event.add_component(alarm_24h_before)
 
-Or even recurrence.
+Or even recurrence, either from a dictionary or a string.
+Note that if you want to add the reccurence rule from a string, you must use the vRecur property. Otherwise the rule will be escaped, making it invalid.
 
 .. code-block:: pycon
 
-    >>> event.add('rrule', {'freq': 'daily'})
+    >>> from icalendar import vRecur
+
+    >>> event.add('rrule', {'freq': 'daily', 'interval': 10})
+
+    >>> event.add('rrule', vRecur.from_ical('FREQ=DAILY;INTERVAL=10'))
```

Comment:
Oh, I found it... We actually run the documentation code to make sure it works.
You add another RRULE entry and that makes the output different a few lines further down.

I think, a good place to add the documentation for this is actually the class string:

https://github.com/collective/icalendar/blob/4725c1bd57ca3afe61e7d45805a2b337842fbd29/src/icalendar/prop.py#L1102

There is almost nothing and more is appreciated! Could you add these two examples to the class string? Could you remove the duplicate entry in the docs?


----

This is another issue to make this class nicer to understand:
https://github.com/collective/icalendar/issues/742
That is - if you like this class and making it pretty :heart:

Reply:
Done. I copied most of the docstring from the rfc. Let me know if you want me to adjust anything.